x-pack/plugins/cases/public/files/index.ts

```diff
@@ -17,6 +17,11 @@ const buildFileKind = (config: FilesConfig, owner: Owner): FileKindBrowser => {
     id: constructFileKindIdByOwner(owner),
     allowedMimeTypes: config.allowedMimeTypes,
     maxSizeBytes: config.maxSize ?? MAX_FILE_SIZE,
+    managementUiActions: {
+      delete: {
```

Comment:
The Files service allows to you define a `reason`. Can you add `reason: The file is managed by Cases.`?